source/MaterialXGenMdl/MdlShaderGenerator.cpp

```diff
@@ -287,14 +287,6 @@ string MdlShaderGenerator::getUpstreamResult(const ShaderInput* input, GenContex
 {
     const ShaderOutput* upstreamOutput = input->getConnection();
 
```

Comment:
Remove bypass to make texture input inlined which is invalid currently in OV. Also exposes input as modifiable which it should be.